src/runtime_src/core/tools/xbmgmt2/flash/flasher.cpp

```diff
@@ -94,7 +94,7 @@ Flasher::E_FlasherType Flasher::getFlashType(std::string typeStr)
  * upgradeFirmware
  */
 int Flasher::upgradeFirmware(const std::string& flasherType,
-    firmwareImage *primary, firmwareImage *secondary)
+    firmwareImage *primary, firmwareImage *secondary, firmwareImage *stripped)
```

Comment:
If this function doesn't change primary, secondary, and stripped, then it should say so by using const.